pkg/isb/stores/jetstream/reader.go

```diff
@@ -270,6 +270,23 @@ func (jr *jetStreamReader) Ack(_ context.Context, offsets []isb.Offset) []error
 	return errs
 }
 
+func (jr *jetStreamReader) NoAck(_ context.Context, offsets []isb.Offset) {
+	done := make(chan struct{})
+	wg := &sync.WaitGroup{}
+	for _, o := range offsets {
+		wg.Add(1)
+		go func(o isb.Offset) {
+			defer wg.Done()
+			o.NoAck()
+		}(o)
+	}
+	go func() {
+		wg.Wait()
+		close(done)
+	}()
+	<-done
```

Comment:
```suggestion
	wg.Wait()
```

Won't this do? Do we need go routine? You can avoid the done channel too.